package/src/components/MessageList/__tests__/MessageList.test.js

```diff
@@ -285,7 +282,6 @@ describe('MessageList', () => {
 
     await waitFor(() => {
       expect(queryAllByTestId('message-system')).toHaveLength(0);
-      expect(queryAllByTestId('error-notification')).toHaveLength(0);
```

Comment:
These tests were flaky so we decided to remove them. Not immediately sure why they started becoming flaky with the upgrade, but likely something to do with `@testing-library/react-native`.